addons/microsoft_calendar/models/calendar.py

```diff
@@ -272,6 +289,13 @@ def _microsoft_to_odoo_values(self, microsoft_event, default_reminders=(), defau
             'show_as': 'free' if microsoft_event.showAs == 'free' else 'busy',
             'recurrency': microsoft_event.is_recurrent()
         })
+        # Edge case: do not add 'start' and 'stop' variables when set by Microsoft while they are the same as in Odoo.
+        # Otherwise, the return will be considered as 'update_time' in 'write' and set attendees state as 'needsAction'.
+        update_edge_case = self.env.context.get('update_edge_case', False)
+        if update_edge_case and self.start == start and self.stop == stop:
+            values.pop('start')
+            values.pop('stop')
```

Comment:
I had a similar case in google_calendar and solved it by not setting start/stop in the `_odoo_values` in google calendar (`_microsoft_to_odoo_values` in microsoft) https://github.com/odoo/odoo/pull/131966/files#diff-7d087533ebb0faba0b7d853b15b2ca5f9e3113bdd9b64c0eea14d14229bf1002R123-R126.  Maybe it's better to fix it like that rather than having to pass stuff in the context.